collect_app/src/main/res/values/strings.xml

```diff
@@ -506,7 +507,7 @@
     <string name="qrcode_without_passwords">This QR code does not contain your admin or server passwords. \n(Tap to add the passwords)</string>
     <string name="qrcode_with_admin_password">This QR code includes your admin password. Please treat it carefully. \n(Tap to remove the password)</string>
     <string name="qrcode_with_server_password">This QR code includes your server password. \n(Tap to remove the passwords)</string>
-    <string name="import_export_settings">Configure via QR Code</string>
+    <string name="import_export_settings">Configure via QR code</string>
```

Comment:
You can remove this resource and use `import_settings` instead. In fact, I recommend renaming that resource to `configure_via_qr_code`.